src/app/login-page/login-page.component.scss

```diff
@@ -0,0 +1,38 @@
+.loginCont{
+  display: flex;
```

Comment:
According to the slides, it should be centered centered on the page.

Reply:
Fixed